pkg/registry/scheduling/rest/storage_scheduling.go

```diff
@@ -106,7 +107,12 @@ func AddSystemPriorityClasses() genericapiserver.PostStartHookFunc {
 				_, err := schedClientSet.PriorityClasses().Get(pc.Name, metav1.GetOptions{})
 				if err != nil {
 					if apierrors.IsNotFound(err) {
-						_, err := schedClientSet.PriorityClasses().Create(pc)
+						// TODO: Remove this explicit conversion after scheduling api move to v1
```

Comment:
these are v1 in https://github.com/kubernetes/kubernetes/pull/73557

scheduling.SystemPriorityClasses() should probably return versioned objects (and the method might need to move to a different package to be able to do that)

Reply:
fixing in this thread https://github.com/kubernetes/kubernetes/pull/76362